src/platforms/android/configuration/integrations/jetpack-compose.mdx

```diff
@@ -0,0 +1,164 @@
+---
+title: Jetpack Compose
+caseStyle: camelCase
+supportLevel: production
+sdk: sentry.java.compose
+description: "Learn more about the Sentry Compose integration."
+categories:
+  - mobile
+---
+
+The `sentry-compose` library provides [Jetpack Compose](https://developer.android.com/jetpack/androidx/releases/compose) Navigation support for Sentry using the [SentryNavigationIntegration](https://github.com/getsentry/sentry-java/blob/524ee49b212c3f2eead20960c3c6825ccdbf8007/sentry-compose/src/androidMain/kotlin/io/sentry/compose/SentryNavigationIntegration.kt). The source can be found [on GitHub](https://github.com/getsentry/sentry-java/blob/main/sentry-compose/src/androidMain/kotlin/io/sentry/compose/).
+
+On this page, we get you up and running with Sentry's Compose Navigation Integration, so that it will automatically add a breadcrumb and start an idle transaction for each navigation event.
```

Comment:
```suggestion
On this page, we get you up and running with Sentry's Compose Navigation Integration, so that it will automatically add a breadcrumb and start a transaction for each navigation event.
```